src/calculateRentalCost.js

```diff
@@ -29,6 +29,29 @@
 
 function calculateRentalCost(days) {
   // write code here
+  let totalPrice = 0;
```

Comment:
3. [checklist] Prefer const over let where possible, to avoid unintentional changes.
for example:
const totalPrice = priceOfRentPerDay * days;

Reply:
fixed, no need for this variable as well - we can return result in even one expression